### PR TITLE
docs: add recommended requirements, roadmap

### PR DESCRIPTION
**Pull request by ralgozino**
- add recommended hardware requirements to the Architecture section in the readme for a standard production-grade installation.
- add roadmap file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,38 @@ The standard way to deploy KFD is to:
 - Deploy all the [Core Modules](#core-modules-) of the distribution
 - Deploy (if needed) any of the [Addon modules](#add-on-modules-)
 
+### Minimum Hardware Requirements
```
Comment (nutellinoit) [inline README.md:52]:
```suggestion
### Recommended Hardware Requirements
```

File: README.md [hunk 2]
```diff
@@ -49,6 +49,38 @@ The standard way to deploy KFD is to:
...
+| Load Balancer  | 2           | 2        | 50        | 2    |
+
+#### Storage
+
+Some modules rely on persistent storage via PersistentVolumeClaims, by default (but configurable) the following capacity will be used:
+
+| Description                                | Size (GB) |
+| ------------------------------------------ | --------- |
+| Prometheus (metrics storage)               | 150       |
+| MinIO Monitoring (metrics storage, 20GBx3) | 60        |
```
Comment (nutellinoit) [inline README.md:78]:
```suggestion
| MinIO Monitoring (metrics storage, 20GBx6) | 120        |
```

File: README.md [hunk 3]
```diff
@@ -49,6 +49,38 @@ The standard way to deploy KFD is to:
...
+| MinIO Logging (logs storage, 20GBx3)       | 60        |
```
Comment (nutellinoit) [inline README.md:79]:
```suggestion
| MinIO Logging (logs storage, 20GBx6)       | 120        |
```

File: README.md [hunk 4]
```diff
@@ -49,6 +49,38 @@ The standard way to deploy KFD is to:
...
+| OpenSearch (logs storage)                  | 30        |
+| MinIO Tracing (traces storage)             | 60        |
```
Comment (nutellinoit) [inline README.md:81]:
```suggestion
| MinIO Tracing (traces storage)             | 120        |
```

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Some changes, (to be double checked) 👇

**Comment by ralgozino** [comment]
> Some changes, (to be double checked) 👇

Updated values (double checked that minio is a statefulset, so your correction was spot-on) and recalculated total

**Review by nutellinoit — APPROVED** [review]
LGTM!

**Pull request merged**